### ISSUE TITLE
[BUG] [Linter] Closing tag </script> reported as orphan

### ISSUE THREAD

**Issue by dyve** [Riverside-Healthcare/djLint#142]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: macOS 12.0.1
 - Python Version: 3.8.12
 - djLint Version: 0.6.1

## Issue
HTML `script` closing tag reported as orphan.

## How To Reproduce

Lint this template:

```
{% load static %}
<script src="{% static "folder/foo.js" %}?version={% some_version %}"></script>
```

Note that the `</script>` (closing) tag is reported as an orphan.
```
H025 2:71 Tag seems to be an orphan. </script>
```

## Configuration

```
# pyproject.toml
[tool.djlint]
# Reference: https://djlint.com/djlint/rules.html
profile = "django"
ignore = "H017,H019,T003,T002,H023,H014,H008,T028,H021,T001,H006,H013,H022"
blank_line_after_tag = "load,extends,include"
max_line_length = 120
```

**Comment by dyve** [comment]
Thanks @christopherpickering

**Comment by christopherpickering** [comment]
no problem, please keep creating issues :)

**Comment by dyve** [comment]
Will do. Have thought about creating a "black for templates" multiple times, but decided that it was not for me. Happy to contribute though. Hope that this becomes a huge success. I found djlint when looking for Django template reformatting for VS Code.

**Comment by christopherpickering** [comment]
Thanks, yeah, it works for me at least - I couldn't find anything decent either. Templates are very tedious to link and format as there are so many different ways to do the same thing. PR's are more than welcome!
